### DISCUSSION TITLE
[Question][Hw4] Issue with `dune-project` file

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs524-program-analysis#383] [Q&A]
Before starting HW5, I wanted to fix problems in HW4, so I ran the added test cases with `make all` in the given Docker image, but got the following error.

```bash
File "test/dune", line 18, characters 3-81:
18 |    (pipe-stdout
19 |     (run ../analyzer interval example1.ll)
20 |     (run cut -f1 -d,)))))
Error: 'pipe-stdout' is only available since version 2.7 of the dune
language. Please update your dune-project file to have (lang dune 2.7).
```

Changing the dune version in `dune-project` to 2.9 (the version from `dune --version` fixed the problem. Was this done when grading HW4? I assume the TAs would have made this change, but I just wanted to be sure :)

**Comment by hyunsukimsokcho** [comment]
Sorry for the inconvenience. As the error msg says, changing the `dune-project` file to indicate dune version 2.7 or further would fix. When grading, we considered the issue.

**Comment by Lee-Janggun** [comment]
Thank you for the confirmation!
